docs/messaging/messaging-spans.md

```diff
@@ -512,6 +513,51 @@ flowchart LR;
 | `messaging.message.id` | `"a1"` | `"a2"` | |
 | `messaging.batch.message_count` |  |  | 2 |
 
+### Batch publishing
+
+Given is a publisher that publishes a batch with two messages to a topic "Q" on
+Kafka, and two different consumers receiving one of the messages.
+
+```mermaid
+flowchart LR;
+  subgraph PRODUCER
+  direction TB
+  P[Span Publish]
+  CA[Span Create A]
+  CB[Span Create B]
+  end
+  subgraph CONSUMER1
+  direction TB
+  D1[Span Receive A]
+  end
+  subgraph CONSUMER2
+  direction TB
+  D2[Span Receive B]
+  end
+  P-- parent -->CA;
+  P-- parent -->CB;
+  CA-. link .-D1;
+  CB-. link .-D2;
+
+  classDef normal fill:green
+  class P,CA,CB,D1,D2 normal
+  linkStyle 0,1,2,3 color:green,stroke:green
+```
+
+| Field or Attribute | Span Publish | Span Create A | Span Create B | Span Receive A | Span Receive B |
+|-|-|-|-|-|-|
+| Span name | `Q publish` | `Q create` | `Q create` | `Q receive` | `Q receive` |
+| Parent |  | Span Publish | Span Publish | | |
+| Links |  |  |  | Span Create A | Span Create B |
+| SpanKind |  | `PRODUCER` | `PRODUCER` | `CONSUMER` | `CONSUMER` |
+| Status | `Ok` | `Ok` | `Ok` | `Ok` | `Ok` |
```

Comment:
nit: we never set status to `OK` in other semconv, but leave it unset, so maybe we should just remove this row since it's not strongly related to the example?